packages/storybook-utils/src/style.css

```diff
@@ -1,4 +1,4 @@
-@import url("sit-onyx/themes/onyx.css");
+@import url("../../sit-onyx/src/styles/variables/themes/onyx.css");
```

Comment:
This works in our local monorepo but will break when used in a real project since the relative path is not resolvable then...we need to import from `sit-onyx` instead